Neos.Media/Classes/Command/MediaCommandController.php

```diff
@@ -338,15 +367,87 @@ public function renderThumbnailsCommand(int $limit = null, bool $quiet = false)
                 break;
             }
         }
-        !$quiet && $this->output->progressFinish();
+    }
+
+    /**
+     * Render asset variants
+     *
+     * Loops over missing configured asset variants and renders them. Optional ``limit`` parameter to
+     * limit the amount of variants to be rendered to avoid memory exhaustion.
+     *
+     * If the re-render parameter is given, any existing variants will be rendered again, too.
+     *
+     * @param integer $limit Limit the amount of variants to be rendered to avoid memory exhaustion
+     * @param bool $quiet If set, only errors will be displayed.
+     * @param bool $recreate
```

Comment:
Missing description for that param. I assume it could be sth like:
```suggestion
     * @param bool $recreate If set, existing asset variants will be re-generated and replaced
```
but I'm merely guessing :)